### PR TITLE
Tutorial: Rate Limiting a service with Kong Gateway

### PR DESCRIPTION
**Pull request by cloudjumpercat**
Preview URL: https://deploy-preview-43--kongdeveloper.netlify.app/tutorials/add-rate-limiting-to-a-service-with-kong-gateway/

- Revised the existing RL service tutorial to clean up the prereqs
- Made some metadata for prereqs, it doesn't work and I'm not sure how to fix it, but hopefully that gives a starting point for what we could do
- Added links to relevant pages to create a better user journey to find this content

### CODE REVIEW

**Comment by cloudjumpercat** [comment]
### My testing experience notes:

_Disclaimer: I've used decK like once, so I'm very new._

- Used the quickstart to quickly get Kong, no problems. Already had Docker and curl.
- I needed a service and a route. I used the rendered examples [here](https://deploy-preview-42--kongdeveloper.netlify.app/tutorials/add-rate-limiting-to-a-service-with-kong-gateway/#:~:text=Kong%20Gateway%20Ready-,Create%20a%20service,-_format_version%3A%20%22) (since they aren't rendering in my prereqs because I've got the formatting wrong).
  - These aren't commands, what do I do with these? Looking at https://docs.konghq.com/deck/1.38.x/guides/getting-started/#export-the-configuration I realized these are part of the kong.yaml config. I feel like it was too hard to run these on their own separately, so I just copied them into my kong.yaml file.
  - Note: I did fix the weird `-` formatting when I copy/pasted that Lucie reported.
- While I was at it, I also copied in the enabling the RL plugin into the yaml file just so it was easier.
- Okay, I have my file edited, now how do I push my changes? Found this section: https://docs.konghq.com/deck/1.38.x/guides/getting-started/#diff-and-sync-the-configuration-to-kong-gateway
  - diff worked just fine
  - ran into errors with the host when I ran sync
- Summary of errors and what I fixed to make it work:
  - `services` needed a host, or so Terminal told me. I added a host before the service name.
  - The prereqs made me think that the `routes` needed to be on a separate line from `services`, when really, what worked for me was making `routes` part of the `services`, for example:
  ```
  _format_version: "3.0"
  services:
    - host: example.com
      name: example_service
      routes:
        - name: example_route
  ```
  - I also continued to get errors until I added a `path` to the route. I used this to help me figure out the right `routes` formatting: https://docs.konghq.com/deck/1.38.x/guides/defaults/#test-default-value-handling

What I originally had for the formatting and thought it had to be (which didn't work): 
```
_format_version: "3.0"
services:
  - name: example_service
routes:
  - name: example_route
    service:
      name: example_service
plugins:
  - name: rate-limiting
    service: example_service
    config:
      second: 5
      hour: 1000
      policy: local
```

File: app/_tutorials/add-rate-limiting-to-a-service-with-kong-gateway.md [hunk 1]
```diff
@@ -32,61 +32,27 @@ tags:
...
-data:
-  name: example_route
-  service:
-    name: example_service
-{% endentity_example %}
-{% endcapture %}
-{{ step | indent: 3 }}
-
-1. Enable the Rate Limiting Plugin on the Service
+1. Enable the Rate Limiting Plugin on the service:
```
Comment (cloudjumpercat) [inline app/_tutorials/add-rate-limiting-to-a-service-with-kong-gateway.md:55]:
```suggestion
1. Enable the Rate Limiting Plugin on the service by adding the following to your `kong.yaml` file:
```

File: app/_tutorials/add-rate-limiting-to-a-service-with-kong-gateway.md [hunk 2]
```diff
@@ -105,9 +71,22 @@ variables:
...
 {{ step | indent: 3 }}
 
-1. Validate
+1. Run the diff command:
+
+  ```bash
+  deck gateway diff kong.yaml
+  ```
+
+  You should see decK reporting that the properties you had changed in the file are going to be changed by decK in Kong Gateway’s database.
```
Comment (cloudjumpercat) [inline app/_tutorials/add-rate-limiting-to-a-service-with-kong-gateway.md:80]:
```suggestion
```

**Comment by mheap** [comment]
> However, how will this work when we aren't only using decK? I don't know if it's safe to assume people know to run those. I'd like to mention that the config goes in a kong.yaml file, but again, that would prevent the step instructions from being reused for Admin API, Konnect UI, etc.

We can assume that tutorials are deck only for now. We may revisit in the future, but it's safe to add deck specific instructions

**Comment by fabianrbz** [comment]
cherry-picked changes in https://github.com/Kong/developer.konghq.com/commit/620320ed86287be34c8a556ad0bcc31ed47ae6fc

**Pull request closed without merging**